tests/test_upload_to_ENA.py

```diff
@@ -130,4 +133,44 @@ def test_single_dry_upload_xml_files_to_ena(self):
                                       'endpoint:')
             mock_info.assert_any_call('file: ELOAD_1.SingleSubmission.xml')
             mock_post.assert_not_called()
-            mock_get.assert_not_called()
\ No newline at end of file
+            mock_get.assert_not_called()
+
+    def test_upload_FTP(self):
+        # Need to use real ENA credentials in submission_config.yml for this test to work
+        assert True
+        return
+        filename = 'test_vcf_file.vcf'
+        with HackFTP_TLS() as ftps:
+            # Connect to delete the file
+            ftps.context.set_ciphers('DEFAULT:@SECLEVEL=1')
+            ftps.connect(cfg.query('ena', 'ftphost'), port=int(cfg.query('ena', 'ftpport', ret_default=21)))
+            ftps.login(cfg.query('ena', 'username'), cfg.query('ena', 'password'))
+            ftps.prot_p()
+            list_files = ftps.nlst()
+            if self.uploader.eload in list_files:
+                ftps.cwd(self.uploader.eload)
+                list_files = ftps.nlst()
+                if self.uploader.eload in list_files:
+                    ftps.delete(filename)
```

Comment:
Not a big deal, but to me this makes more sense as a clean-up method run after the test, rather than as a set-up method run before...

Reply:
You are right but since I need the ENA credential to access the FTP I could not run this in a `setUp` or `tearDown`. And within the test I'd rather do it at the start in case the previous test failed. 
I know it's very backward ...